tests/test_utils.py

```diff
@@ -230,3 +232,19 @@ def test_format_timestamp(timestamp, local_timezone, expected, unstub):
     assert formatted == expected
 
     unstub()
+
+
+def test_timestamp_before_now():
+    # future timestamp
+    future_timestamp = str(datetime.now() + timedelta(days=1))
```

Comment:
It'd be helpful to show here (in a comment or docstring) that these strings match what the service returns

Reply:
yea i can do that.  In the end it doesnt actually matter what the service returns because the `format_timestamp` converts it to a datetime object and datetime is what we use to convert all date responses from the service.  So using datetime to create test data will always "match" what we get from the service.